src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorAbsSendTime.java

```diff
@@ -0,0 +1,627 @@
+package org.jitsi.impl.neomedia.rtp.remotebitrateestimator;
+
+import org.ice4j.util.RateStatistics;
+import org.jitsi.impl.neomedia.codec.audio.silk.K2aFLP;
+import org.jitsi.impl.neomedia.transform.PacketTransformer;
+import org.jitsi.service.neomedia.RawPacket;
+import org.jitsi.service.neomedia.rtp.RemoteBitrateEstimator;
+import org.jitsi.util.Logger;
+
+import javax.media.rtp.RTPHeader;
+import java.util.*;
+
+/**
+ * webrtc.org abs_send_time implementation as of June 26, 2017.
+ * @author Julian Chukwu
+ */
+public class RemoteBitrateEstimatorAbsSendTime
+    implements RemoteBitrateEstimator
+{
+    private static final Logger logger = Logger
+            .getLogger(RemoteBitrateEstimatorAbsSendTime.class);
+
+    private enum probeResult
```

Comment:
The convention is to name classes and enums with the first letter capitalized (ProbeResult)